tests/unit/test_routes.py

```diff
@@ -32,6 +34,19 @@ def test_get_resources(module_client, module_db):
         assert (resource.get('category'))
         assert (isinstance(resource.get('languages'), list))
 
+    ua = datetime.now() + timedelta(days=-7)
+    uaString = ua.strftime('%m-%d-%Y')
+    response = client.get(f"/api/v1/resources?updated_after={uaString}")
+    assert (response.status_code == 200)
+
+    # If updated_after date is after today, then should return a 422
+    ua = datetime.now() + timedelta(days=1)
+    uaString = ua.strftime('%m-%d-%Y')
+    response = client.get(f"/api/v1/resources?updated_after={uaString}")
+    assert (response.status_code == 422)
+    assert (response.json.get('errors')[0].get('code') == "unprocessable-entity")
+    assert ("updated_after" in response.json.get('errors')[0].get('message'))
```

Comment:
This is tieing the implementation to the behavior too strongly. We want to _eventually_ test that on an error, we call the standard error handler. Not that the value returned is the correct format. For that we'd want to use approval tests.

Reply:
Ok, so should I just get rid of that assertion altogether? If not, how would you change it?

Reply:
It's your call, just want to be clear what you are testing.